app/controllers/comments_controller.rb

```diff
@@ -10,10 +10,13 @@ def create
 
     if @comment.valid?
       @comment.save
-      redirect_to user_post_path(id: @post.id, user_id: @user.id)
+      # redirect_to user_post_path(id: @post.id, user_id: @user.id)
+      flash.notice = 'Comment added successfuly.'
```

Comment:
- [ ] Kindly redirect to the post#show when the comment has been created because is where your flash messages should be rendered. hint :point_down: 

```.rb
 if @comment.valid?
      @comment.save
      # redirect_to user_post_path(id: @post.id, user_id: @user.id)
      flash.notice = 'Comment added successfuly.'
      redirect_to user_post_path(@user.id, @post.id)
    else
........

``